code/datums/diseases/black_goo.dm

```diff
@@ -141,6 +141,8 @@
 
 /datum/disease/black_goo/proc/zombie_transform(mob/living/carbon/human/human)
 	set waitfor = 0
+	if(human.buckled)
+		human.buckled.unbuckle()
```

Comment:
This would look a bit better if it happened after the `sleep()`s, I think.

```diff
/datum/disease/black_goo/proc/zombie_transform(mob/living/carbon/human/human)
	set waitfor = 0
-	if(human.buckled)
-		human.buckled.unbuckle()
	zombie_is_transforming = TRUE
	human.vomit_on_floor()
	human.adjust_effect(5, STUN)
	sleep(20)
	human.make_jittery(500)
	sleep(30)
	if(human && human.loc)
+		if(human.buckled)
+			human.buckled.unbuckle()
		if(human.stat == DEAD)
```
<hr>

**Current:** (Unbuckles, plays a little animation, then transforms)

https://github.com/cmss13-devs/cmss13/assets/57483089/192652af-fbc1-42db-a411-546159da1aa2

**Proposed Change:** (Plays a little animation, unbuckles, then transforms)

https://github.com/cmss13-devs/cmss13/assets/57483089/4fd9a358-ca1c-4a5d-b251-2904ab7ba1ad